procedures/auxtel/observation_procedures/Parabolic_Focus_Sequence.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>Instead of saving a file, I've sent the sequence information into the script log to show up in the EFD <code>lsst.sal.Script.logevent_logMessage</code>  <span class="ql-cursor">﻿</span></p>

Reply:
<p><code>script_log.log.info(f'END -- Focus Determination {filter_to_focus}+{grating_to_focus} \n'</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;f'Sequences -- {parabolic_focus_data} \n'</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;f'Time UTC -- {astropy.time.Time(astropy.time.Time.now())}') </code> </p>

Reply:
<p>I think that's a good idea for the moment. </p>